ukaea/ditto
# Feature/add metadata to archive files/#8

Add functionality so that whenever a file transfer occurs the files transferred are listed in the .ditto_archived file. This file is never transferred via ditto and is updated with each new transfer to show the most recent state of each file transferred.

## To test:
* Run the system tests
   - load vm
   - `cd systemTests/testScenarios`
   - `pytest -s`
* Run unit tests for internal data service and archiver
* Run unit tests for data replication service to ensure archiver is being called

### Stories covered
#7 #8
